modules/nw-external-dns-operator.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+// * networking/understanding-external-dns-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-dns-operator_{context}"]
+= External DNS Operator
+
+The External DNS Operator implements the External DNS API from the `olm.openshift.io` API group. The External DNS Operator deploys the `ExternalDNS` using a deployment. The `external-dns-operator` deployment watches the resources such as services and routes in the cluster and updates the External DNS providers.
+
+.Procedure
+
+You can deploy the ExternalDNS Operator on demand from the OperatorHub with a `Deployment` object.
+
```

Comment:
We can check the install plan first:
```
$ oc -n external-dns-operator get sub external-dns-operator -o yaml | yq '.status.installplan.name'
install-zcvlr
$ oc -n external-dns-operator get ip install-zcvlr -o yaml | yq .status.phase'
Complete
```
It has to be `Completed`. And the we can check the `Deployment` resource.
In case of a problem we can suggest to go to [OLM troubleshooting](https://olm.operatorframework.io/docs/tasks/troubleshooting/) if this conforms to the rules of OCP docs.

Reply:
Ack'ed. Added the commands. Still checking about including https://olm.operatorframework.io/docs/tasks/troubleshooting/ into the docs

Reply:
> @quarterpin : `service` based ExtDNS configuration are indeed not part of the samples but the service type is well described in [the configuration page](https://deploy-preview-45514--osdocs.netlify.app/openshift-enterprise/latest/networking/external_dns_operator/nw-external-dns-operator-configuration-parameters.html). I think it's fine to be more OpenShift oriented and use routes.
> 
> The part about checking the logs of the operand deployment sounds more like a troubleshooting chapter. Don't know if the troubleshooting is a part of the OCP docs, what do you think @xenolinux?

+1, We can pass on including the troubleshooting commands. We have included the section for checking the External DNS Operator logs, IMO that should suffice.